### PR TITLE
Realsight

### PR DESCRIPTION
**Pull request by realsightAPM**
Bayesian Regression Change Point for Anomaly Detection
[README.pdf](https://github.com/numenta/NAB/files/465321/README.pdf)

### CODE REVIEW

**Comment by realsightAPM** [comment]
[Numenta Contributor License V1.1 - signed.pdf](https://github.com/numenta/NAB/files/466336/Numenta.Contributor.License.V1.1.-.signed.pdf)

**Comment by realsightAPM** [comment]
[Numenta Contributor License V1.1 - signed.pdf](https://github.com/numenta/NAB/files/468964/Numenta.Contributor.License.V1.1.-.signed.pdf)

**Comment by BoltzmannBrain** [comment]
@RedShit please push the code that generated these results.

**Comment by realsightAPM** [comment]
@BoltzmannBrain you can check this repo https://github.com/realsightAPM/NABSubmit and modify this file (https://github.com/realsightAPM/NABSubmit/blob/master/src/main/java/com/realsight/brain/model/noiseseries/ChangePoint.java) to generate results. @RedShit  will make it easy to use later.

File: .project [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (oxtopus) [inline .project]: Please remove this file.  I believe it's meant for use with Eclipse and not meant to be shared.

**Comment by marionleborgne** [comment]
Hi @realsightAPM. Please address @oxtopus's [comment](https://github.com/numenta/NAB/pull/268#pullrequestreview-649783) if you would like us to proceed with the PR review. Thanks!

**Comment by marionleborgne** [comment]
Closing this pull request for lack of activity.

**Pull request closed without merging**